### ISSUE TITLE
Flag on users that signifies they have been "blocked"

### ISSUE THREAD

**Issue by jarmstrng** [ifmeorg/ifme#574]
See https://github.com/julianguyen/ifme/issues/14

**Comment by stebbins** [comment]
Added `banned` column to User

**Comment by baohouse** [comment]
@stebbins Is there a PR for this?

**Comment by stebbins** [comment]
@baohouse Our thoughts were to complete the entire project on [this fork](https://github.com/stebbins/ifme) and then PR once the whole thing is completed. Do you think we should do PRs incrementally instead? .efinitely open to feedback.

**Comment by baohouse** [comment]
@stebbins Whenever I see an issue closed, it's either because the task is complete and the feature is in production, or the issue is a duplicate and/or not to be worked on. I just wasn't sure which one was the case when these issues were closed.

**Comment by stebbins** [comment]
@baohouse Fair enough, I will keep them open in the future to avoid confusion. I can track completion via the project board anyways - once it gets moved into production I'll close them all unless you have any objections :smiley:
